scout: view, generate, export sbom (#18105)
docker/docs: content/build/attestations/sbom.md

$ docker buildx build --attest type=sbom,generator=<image> .
```

> **Tip**
>

> > The Docker Scout SBOM generator is available as a BuildKit plugin. See

Reviewer: "BuildKit plugin" does not sound right. I think we can just strip this:

```suggestion
> The Docker Scout SBOM generator is also available. See
```
